tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3009,6 +3081,23 @@ def yuv_to_rgb(images):
 
   Returns:
     images: tensor with the same shape as `images`.
+
+  Usage Example:
+  
+  >>> image = [[[1.0, 2.0, 3.0],[4.0, 5.0, 6.0]],
```

Comment:
pylint might complain about the lack of space after `,` in `],[`. If it does, please fix.